src/coreclr/src/Directory.Build.props

```diff
@@ -1,9 +0,0 @@
-<Project>
-  <Import Project="../Directory.Build.props" />
-
-  <!-- Properties that should be defined for all managed product assemblies -->
-  <PropertyGroup>
-    <Company>Microsoft Corporation</Company>
-    <Product>Microsoft%AE .NET</Product>
-  </PropertyGroup>
```

Comment:
I assume you removed that block entirely as these props are already set by Arcade?

Reply:
Hmm, not really, in fact I overlooked this one ;-); I'll add it to the <code>src/coreclr/Directory.Build.props</code>, thanks for pointing that out!

Reply:
Or are you saying that I actually don't need to set them? I mean, it was not intentional but if it was duplicative, let's just get rid of that.

Reply:
You don't need to add them back. They are already set by the Arcade SDK.